host/versioning_test.go

```diff
@@ -26,9 +26,15 @@ package host
 
 import (
 	"context"
+	"flag"
 	"fmt"
+	"testing"
 	"time"
 
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"go.temporal.io/server/common/dynamicconfig"
```

Comment:
import ordering is all messed up here already, but in theory we standardized on having three groups: 1. stdlib, 2. third-party (including sdk and api), 3. go.temporal.io/server/...